app/src/main/java/io/github/djunicode/salespersontracker/DailyTask.java

```diff
@@ -0,0 +1,62 @@
+package io.github.djunicode.salespersontracker;
+
+public class DailyTask {
+
+    private String Assigned_Date;
+    private String Assigned_Time;
+    private String Quantity;
+    private String Completed;
+    private String Notes;
+    private String Assigned_By;
+    private String Assigned_To;
+    private String Item_Ref;
+    private String id;
```

Comment:
Please use uniform variable naming style. No underscores, no variable names starting with capitals
Example
HttpUrlConnection connection
Order pendingOrder.
You can refactor them easily by using Shift+F6 when cursor is on the variable